Source/WebKit/Platform/IPC/unix/ConnectionUnix.cpp

```diff
@@ -51,7 +51,8 @@
 
 // Although it's available on Darwin, SOCK_SEQPACKET seems to work differently
 // than in traditional Unix so fallback to STREAM on that platform.
-#if defined(SOCK_SEQPACKET) && !OS(DARWIN)
+// SOCK_SEQPACKET isn't available on Haiku.
```

Comment:
It's defined in Haiku headers, but not implemented. Maybe the comment should clarify that, because if it wasn't defined at all, there wouldn't be a problem here.

Reply:
I see what you mean about the comments wording. Hopefully the new wording is better.